lib/std/compress/deflate/compressor_test.zig

```diff
@@ -397,8 +396,7 @@ test "deflate/inflate string" {
     // Skip wasi because it does not support std.fs.openDirAbsolute()
     if (builtin.os.tag == .wasi) return error.SkipZigTest;
```

Comment:
This this skip still stay now that `std.fs.openDirAbsolute()` is no longer used here?